apps/studio/src-commercial/entrypoints/utility.ts

```diff
@@ -151,5 +155,11 @@ async function init() {
   ormConnection = new ORMConnection(platformInfo.appDbPath, false);
   await ormConnection.connect();
 
+  try {
+    await pluginManager.initialize();
```

Comment:
same q?

Reply:
handled in this commit a71dedff196e59a93813058e286bc671c8745d53